$NetBSD: patch-src_apps_deskflow-core_CMakeLists.txt,v 1.1 2026/01/06 19:57:10 tnn Exp $

CMake Error at src/apps/deskflow-core/CMakeLists.txt:41 (install):
  install TARGETS RUNTIME_DEPENDENCY_SET is not supported on system "NetBSD"

--- src/apps/deskflow-core/CMakeLists.txt.orig	2026-01-02 18:59:24.301230442 +0000
+++ src/apps/deskflow-core/CMakeLists.txt
@@ -40,7 +40,6 @@ target_link_libraries(
 
 install(
   TARGETS ${target}
-  RUNTIME_DEPENDENCY_SET coreDeps
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
 )
 
